pkg/apis/autoscaling/register.go

```diff
@@ -97,7 +97,13 @@ const (
 	//   KPA will compute the decay multiplier automatically based on the window size
 	//   and it is at least 0.2. This algorithm might not utilize all the values
 	//   in the window, due to their coefficients being infinitesimal.
-	MetricAggregationAlgorithm = GroupName + "/metricAggregationAlgorithm"
+	MetricAggregationAlgorithmKey = GroupName + "/metricAggregationAlgorithm"
+	// AlgorithmLinear is the linear aggregation algorithm with all weights
+	// equal to 1.
+	AlgorithmLinear = "linear"
```

Comment:
should this be MetricAggregationAlgorithmLinear to be consistent?

Reply:
Too long,but sure :)